clang/test/lit.cfg.py

```diff
@@ -210,6 +210,9 @@ def have_host_clang_repl_cuda():
 config.substitutions.append(("%host_cc", config.host_cc))
 config.substitutions.append(("%host_cxx", config.host_cxx))
 
+# Determine whether the test target is compatible with execution on the host.
+if config.host_arch in config.target_triple:
+    config.available_features.add("host-arch-compatible")
```

Comment:
One nit that could be considered before committing: maybe `host-target-compatible` is slightly more descriptive?

Reply:
Thanks for the suggestion, renamed it